FastSurferCNN/README.md

```diff
@@ -10,66 +10,77 @@ The network was trained with conformed images (UCHAR, 1-0.7 mm voxels and standa
                     By default, the program checks if you have enough memory to run the view aggregation on the gpu. 
                     The total memory is considered for this decision. 
-                    If this fails, or you actively overwrote the check with setting "--run_viewagg_on cpu", view agg is run on the cpu. 
-                    Equivalently, if you define "--run_viewagg_on gpu", view agg will be run on the gpu (no memory check will be done).
-* --device <str>: Device for processing ("cpu","cuda") (optional, default: cuda)
-* --hires: Switch on high resolution processing (no conforming to 1mm, but to smallest voxel size).
-* --batch_size: Batch size for inference. Default=8
+                    If this fails, or you actively overwrote the check with setting `--viewagg_device cpu`, view agg is run on the cpu. 
+                    Equivalently, if you define `--viewagg_device gpu`, view agg will be run on the gpu (no memory check will be done).
+* `--device <str>`: Device for processing ("cpu","cuda") (optional, default: cuda)
+* `--hires`: Switch on high resolution processing (no conforming to 1mm, but to smallest voxel size).
+* `--batch_size`: Batch size for inference. Default=1
 
 
 ### Example Command Evaluation Single Subject
-To run the network on MRI-volumes of subjectX in ./data (specified by --t1 flag; e.g. ./data/subjectX/orig.mgz), change into the *FastSurferCNN* directory and run the following commands: 
+To run the network on MRI-volumes of subjectX in ./data (specified by `--t1` flag; e.g. ./data/subjectX/t1-weighted.nii.gz), change into the *FastSurferCNN* directory and run the following commands: 
 
 ```
-python3 run_prediction.py --t1 ../data/subjectX/orig.mgz \
---sd ../data \
+python3 run_prediction.py --t1 ../data/subjectX/t1-weighted.nii.gz \
+--sd ../output \
 --t subjectX \
---seg_log ../data/temp_Competitive.log \
+--seg_log ../output/temp_Competitive.log \
 --ckpt_sag ../checkpoints/aparc_vinn_sagittal_v2.0.0.pkl \
 --ckpt_cor ../checkpoints/aparc_vinn_coronal_v2.0.0.pkl \
 --ckpt_ax ../checkpoints/aparc_vinn_axial_v2.0.0.pkl 
 ```
 
-The output will be saved in ./data/subjectX/aparc.DKTatlas+aseg.deep.mgz.
+The output will be stored in:
+
+- ../output/subjectX/mri/aparc.DKTatlas+aseg.deep.mgz (large segmentation)
+- ../output/subjectX/mri/mask.mgz (brain mask)
+- ../output/subjectX/mri/aseg_noCC.mgz (reduced segmentation)
+
 Here the logfile "temp_Competitive.log" will include the logfiles of all subjects. If left out, the logs will be written to stdout
 
+
 ### Example Command Evaluation whole directory
 To run the network on all subjects MRI-volumes in ./data, change into the *FastSurferCNN* directory and run the following command: 
 
 ```
 python3 run_prediction.py --in_dir ../data \
---sd ../data \
---seg_log ../data/temp_Competitive.log \
+--sd ../output \
+--seg_log ../output/temp_Competitive.log \
 --ckpt_sag ../checkpoints/aparc_vinn_sagittal_v2.0.0.pkl \
```

Comment:
Same here

Reply:
This is resolved. See [here](https://github.com/Deep-MI/FastSurfer/pull/231)